### PR TITLE
CometChat Prebuilt Extension

### PR DESCRIPTION
**Pull request by chadbailey59**

### CODE REVIEW

**Comment by chadbailey59** [comment]
Still very much a WIP, but it gets us CometChat in an iframe! :)

File: public/extensions/callstate.js [hunk 1]
```diff
@@ -20,8 +20,9 @@ export default {
   updateCallState: function (key, newState, broadcast = true) {
     // used for an extension to request a state update,
     // which will also get broadcast to everybody else on the call.
+    console.log("updating call state: ", key, newState, broadcast);
     applyStateUpdate(key, newState);
-    if (broadcast) {
+    if (broadcast === true) {
```
Comment (jamsea) [inline public/extensions/callstate.js:25]: 👍 This prevents any JavaScript truthiness shenanigans

**Comment by jamsea** [comment]
@chadbailey59 do we want the chat window to close when we toggle the Chat button? Similar to how prebuilt's chat can toggle on and off?

**Comment by chadbailey59** [comment]
@jamsea yes, we definitely do :) I forgot that bug still existed when I pushed code last night so I could put this in front of somebody. I'll take care of it ASAP

**Pull request merged**